### ISSUE TITLE
Multiple independent replications with the same source and destination?

### ISSUE THREAD

**Issue by ndouglas** [couchbase/couchbase-lite-ios#680]
I was just testing something and noticed that if you have e.g. an active pull replication from one database to another, creating and starting a new pull replication seems to stop the first.

This can be inconvenient since the replication filters and so forth can be vastly different between the two, containing an overlapping or non-overlapping set of documents.

Can this be changed or worked around, except (of course) by creating a new replication whose filters combine the criteria of the two "parent" replications?

**Comment by ndouglas** [comment]
See couchbase/couchbase-lite-ios#680.

**Comment by snej** [comment]
> creating and starting a new pull replication seems to stop the first

That shouldn't be happening unless they're identical replications. Could you show an example of the code that creates the two replications?

**Comment by ndouglas** [comment]
Sorry for the delay -- I had a dentist's appointment yesterday :(

I've confirmed with further testing that the replications work as you say.  

>That shouldn't be happening unless they're identical replications.

I now presume I was somehow doing this in some way in my original test, but I can't figure out how.  It's complicated because there was a decent amount of boilerplate code in the test, and I'm specifically working with setting up/stopping replications automatically in response to listeners appearing/disappearing from the network, so there's a decent amount of asynchronous and event-based code in there.

Maybe I should go back to programming in QBASIC :-(

I opened the original ticket, though, because I couldn't find any documentation on the tubes specifying whether multiple replications connecting the same two endpoints could in fact coexist.  Maybe it could be added to the documentation, or maybe it was perfectly clear to everyone but me :-)

Thanks again, @snej... one of these days, I'll open a ticket that's not a waste of your time.
